README.md

```diff
@@ -1,134 +1,461 @@
-# SQLite Foreign Data Wrapper for PostgreSQL
-This PostgreSQL extension is a Foreign Data Wrapper for [SQLite][1].
+SQLite Foreign Data Wrapper for PostgreSQL
+==========================================
 
-The current version can work with PostgreSQL 11, 12, 13, 14 and 15.
+This is a foreign data wrapper (FDW) to connect [PostgreSQL](https://www.postgresql.org/)
+to [SQLite](https://sqlite.org/) database file. This FDW works with PostgreSQL 11, 12, 13, 14, 15 and confirmed with SQLite 3.38.5.
 
-## Installation
-### 1. Install SQLite library
+<img src="https://upload.wikimedia.org/wikipedia/commons/2/29/Postgresql_elephant.svg" align="center" height="100" alt="PostgreSQL"/>	+	<img src="https://upload.wikimedia.org/wikipedia/commons/3/38/SQLite370.svg" align="center" height="100" alt="SQLite"/>
 
-For debian or ubuntu:
-<pre>
-apt-get install libsqlite3-dev
-</pre>
+History of releases and main features see in [CHANGELOG](CHANGELOG).
```

Comment:
Please remove this line and CHANGELOG file. You can see the same information at https://github.com/pgspider/sqlite_fdw/releases

Reply:
Deleted.

Reply:
Thank you for fixing.